app/routes/index.js

```diff
@@ -14,7 +14,7 @@ export default class IndexRoute extends Route {
         credentials: 'include',
       });
       const userData = await response.json();
-      if (response.status === 200 && !userData.incompleteUserDetails) {
+      if (response.status === 200) {
         return userData?.data?.currentStatus?.state ?? defaultStatus;
```

Comment:
Can we remove this `defaultStatus` now??
Because we are handling the case as `Does not Exist`
Let me know your thoughts on this.

Reply:
yeah Right. We can get rid of the `defaultStatus` now as we are asking the user to set it if it doesn't exist.